tests/fakeapi/hello.py

```diff
@@ -410,6 +410,10 @@ def throw_problem_exception():
     )
 
 
+def throw_value_error():
+    raise ValueError('Something really wrong.')
```

Comment:
Does this message get lost?

Reply:
Yes, but this is a user's decision, not connexion's. The exception data get lost when the custom error handler decides to not pass it to `connexion.problem` in any way. In the regression test case scenario this happens here:

```py
def the_error_handler(error):
    return problem(599, 'Problem', 'Something wrong.')
app.add_error_handler(ValueError, the_error_handler)
```

Note that ther `error` parameter is not used inside the function.